daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -66,6 +66,49 @@ npm run start:dapr-http
 
 > ℹ️ **Note:** The `app-port` is required here, as this is where our server will need to bind to. Dapr will check for the application to bind to this port, before finishing start-up.
 
+### Using HTTP (bring your own express webserver)
+
+By default, for sidecar to application communication we bring a webserver that is configured to listen to the Sidecar calls as defined in the Dapr API specification. However, in some scenarios (e.g., when you are building a REST API backend and want to integrate Dapr directly in it) you want to bring your own webserver.
+
+As the Dapr JS SDK team, we do support this scenario, but currently this is only available for [`express`](https://www.npmjs.com/package/express).
```

Comment:
```suggestion
Instead of using the built-in web server for Dapr sidecar to application communication, you can also bring your own instance. This is helpful in scenarios like when you are building a REST API back-end and want to integrate Dapr directly in it.  

Note, this is currently available for [`express`](https://www.npmjs.com/package/express) only.
```